test/e2e/charts/clickhouse-tls/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Settings for ingress
+ingress:
+  apiVersion: networking.k8s.io/v1
+  enable: false
+  enable_https: false
```

Comment:
Helm value names use camelCase,  not snake_case

Reply:
Fixed. Thanks.